setup.py

```diff
@@ -11,7 +11,7 @@
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
 requirements = [
-    'websocket-client >= 0.32.0',
+    'websocket-client >= 0.32.0, < 1.0.0',
     'requests >= 2.14.2, != 2.18.0',
```

Comment:
Can someone take a moment and also fix this? :D Otherwise we are back here in x days or weeks?

```suggestion
    'requests >= 2.14.2, != 2.18.0, < 3',
```